$MirOS: ports/www/php/patches/patch-main_php_open_temporary_file_c,v 1.1.7.1 2005/03/18 16:00:00 tg Exp $
$OpenBSD: patch-main_php_open_temporary_file_c,v 1.1.1.1 2004/10/02 11:32:37 robert Exp $
--- main/php_open_temporary_file.c.orig	Sat Jul 16 12:14:23 2005
+++ main/php_open_temporary_file.c	Wed Jan 25 17:54:43 2006
@@ -123,7 +123,7 @@ static int php_do_open_temporary_file(co
 		trailing_slash = "/";
 	}
 
-	(void)snprintf(opened_path, MAXPATHLEN, "%s%s%sXXXXXX", path, trailing_slash, pfx);
+	(void)snprintf(opened_path, MAXPATHLEN, "%s%s%sXXXXXXXXXX", path, trailing_slash, pfx);
 
 #ifdef PHP_WIN32
 	if (GetTempFileName(path, pfx, 0, opened_path)) {
